doc/Advanced.md

```diff
@@ -935,8 +935,10 @@ should not be present in real configuration files.
         "filterDiscardDebug": "false",    "": "Set to 1 to discard debug and trace messages (default: false)",
         "filterDiscardLevel": "2",        "": "Message at this level or above are discarded (default: 21 - Trace)",
         "filterDiscardFile": "",          "": ["If set, the discarded messages will go to this file (default: <none>)",
-                                              "The keyword _ID_, if used, is replaced by the device ID.", 
-                                              "Discarded Debug messages don't go there."]
+                                              "The keyword _ID_, if used, is replaced by the device ID.",
+                                              "Discarded Debug messages don't go there."],
+        "filterRotateMaxBytes": "",       "": "Maximum size of the discard file.", 
+        "filterRotateMaxFiles": "",       "": "Maximum number of discard files."
```

Comment:
why there could be more than one?

Reply:
isn't that how logrotate usually works ? when a file is full it creates a new one, within the limit of the number of files ?

Reply:
ah yes, of course